x/gov/client/cli/util.go

```diff
@@ -95,7 +95,7 @@ type proposal struct {
 	Summary         string            `json:"summary"`
 	ProposalTypeStr string            `json:"proposal_type,omitempty"`
 
-	proposalType govv1.ProposalType `json:"-"`
+	ProposalType govv1.ProposalType `json:"-"`
```

Comment:
I know `proposal` is private, so the following isn't that much of a problem, but the idea is proposa.proposalType should not be filled in manually, this is why it is private.
I'd rather have us removing the struct annotation here instead.